pandas/core/arrays/integer.py

```diff
@@ -174,8 +174,12 @@ def coerce_to_array(values, dtype, mask=None, copy=False):
         inferred_type = infer_dtype(values)
         if inferred_type not in ['floating', 'integer',
                                  'mixed-integer', 'mixed-integer-float']:
```

Comment:
i don’t recall if we are checking for exactly ndim == 1 right after we convert to ndarray

can u add this as well  ( and some tests)